src/System.Runtime.Serialization.Formatters/src/Resources/Strings.resx

```diff
@@ -129,4 +129,223 @@
+  </data>
+  <data name="Serialization_UnableToFixup" xml:space="preserve">
+    <value>Cannot perform fixup.</value>
+  </data>
+  <data name="ArgumentOutOfRange_ObjectID" xml:space="preserve">
+    <value>objectID cannot be less than or equal to zero.</value>
+  </data>
+  <data name="Serialization_RegisterTwice" xml:space="preserve">
+    <value>An object cannot be registered twice.</value>
+  </data>
+  <data name="Serialization_NotISer" xml:space="preserve">
+    <value>The given object does not implement the ISerializable interface.</value>
+  </data>
+  <data name="Serialization_ConstructorNotFound" xml:space="preserve">
+    <value>The constructor to deserialize an object of type '{0}' was not found.</value>
+  </data>
+  <data name="Serialization_ObjectNotSupplied" xml:space="preserve">
+    <value>The object with ID {0} was referenced in a fixup but does not exist.</value>
+  </data>
+  <data name="Serialization_TypeLoadFailure" xml:space="preserve">
+    <value>Unable to load type {0} required for deserialization.</value>
+  </data>
+  <data name="Serialization_IncorrectNumberOfFixups" xml:space="preserve">
+    <value>The ObjectManager found an invalid number of fixups. This usually indicates a problem in the Formatter.</value>
+  </data>
+  <data name="Serialization_InvalidFixupType" xml:space="preserve">
+    <value>A member fixup was registered for an object which implements ISerializable or has a surrogate. In this situation, a delayed fixup must be used.</value>
+  </data>
+  <data name="Serialization_IdTooSmall" xml:space="preserve">
+    <value>Object IDs must be greater than zero.</value>
+  </data>
+  <data name="Serialization_ParentChildIdentical" xml:space="preserve">
+    <value>The ID of the containing object cannot be the same as the object ID.</value>
+  </data>
+  <data name="Serialization_InvalidType" xml:space="preserve">
+    <value>Only system-provided types can be passed to the GetUninitializedObject method. '{0}' is not a valid instance of a type.</value>
+  </data>
+  <data name="Arg_ArrayZeroError" xml:space="preserve">
+    <value>Array must not be of length zero.</value>
+  </data>
+  <data name="Argument_MustSupplyParent" xml:space="preserve">
+    <value>When supplying the ID of a containing object, the FieldInfo that identifies the current field within that object must also be supplied.</value>
+  </data>
+  <data name="Argument_MemberAndArray" xml:space="preserve">
+    <value>Cannot supply both a MemberInfo and an Array to indicate the parent of a value type.</value>
+  </data>
+  <data name="Serialization_CorruptedStream" xml:space="preserve">
+    <value>Invalid BinaryFormatter stream.</value>
+  </data>
+  <data name="Serialization_HeaderReflection" xml:space="preserve">
+    <value>Header reflection error: number of value members: {0}.</value>
+  </data>
+  <data name="ArgumentNull_WithParamName" xml:space="preserve">
+    <value>Parameter '{0}' cannot be null.</value>
```

Comment:
Nit: Same with this resource string. It should probably deleted. `ArgumentNullException` already includes the param name in the default message.